### DISCUSSION TITLE
anyone experiencing slow with visual editor

### DISCUSSION THREAD

**Discussion by lidaof** [requarks/wiki#4916] [Help / Questions]
My visual editor somehow is pretty slow while trying to edit something. anyone has similar issue? Thanks!
running Wiki.js 2.5.214 on a Linux Redhat server

**Comment by wupasscat** [comment]
Mine is too. Did you find a solution?
Wiki.js 2.5.285 on Docker

**Reply by lidaof** [reply]
nope...still having this issue...
